.github/workflows/nightly-check-db.yml

```diff
@@ -0,0 +1,35 @@
+name: nightly-check-db
+
+on:
+  schedule:
+    - cron: '2 14 * * * *'
+
+env:
+  PYTHONPATH: ${{ github.workspace }}
+
+jobs:
+  download-snapshot:
+    runs-on: ubuntu-latest
+    container: quarkchaindocker/pyquarkchain:mainnet1.1.1
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: create workspace
+      run: mkdir -p tmp/workspace
+
+    - name: Install Project Dependencies
+      run: PYTHONPATH=/code/pyquarkchain pip3 install -e .
+      
+    - name: Download DB snapshot
```

Comment:
this step does too many things, separate it to two parts, 1 for downloading and 1 for running the check-db procedures

Reply:
done